dumb-init.spec

```diff
@@ -0,0 +1,46 @@
+Name:           dumb-init
+Version:        1.1.3
+Release:        1%{?dist}
+Summary:        Entry-point for containers that proxies signals
+
+License:        MIT
+URL:            https://github.com/Yelp/dumb-init
+Source0:        https://github.com/Yelp/dumb-init/archive/v1.1.3.tar.gz
+
+BuildRequires:  vim-common
+BuildRequires:  help2man
+
+%description
+dumb-init is a simple process supervisor and init system designed to run as
+PID 1 inside minimal container environments (such as Docker).
+
+* It can handle orphaned zombie processes.
+* It can pass signals properly for simple containers.
+
+%prep
+%setup -q 
+
+%build
+
+# if we are building a release then this is not needed
+# make VERSION.h 
+gcc -std=gnu99 %{optflags} -o %{name} dumb-init.c 
```

Comment:
no, fedora has quality guidelines and they require it to be dynamically linked.
they do this to go-lang binaries too like etcd. Static linking might need the approval of Fedora Engineering Steering Committee (FESCo) because its security concerns
we might if you wish provide a "dumb-init-static" subpackage.

https://fedoraproject.org/wiki/Packaging:Guidelines#Statically_Linking_Executables

I have opened a review ticket 

https://bugzilla.redhat.com/show_bug.cgi?id=1367033